### PR TITLE
millennium error

### PR DESCRIPTION
**Pull request by Bettman66**
## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on Tasmota core ESP8266 V.2.7.2.1
  - [ ] The code change is tested and works on core ESP32 V.1.12.2
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by Bettman66** [comment]
Tuya does not use the millennium

File: tasmota/xdrv_16_tuyamcu.ino [hunk 1]
```diff
@@ -808,7 +808,7 @@ void TuyaSetTime(void) {
   uint16_t payload_len = 8;
   uint8_t payload_buffer[8];
   payload_buffer[0] = 0x01;
-  payload_buffer[1] = (uint8_t)RtcTime.year;
+  payload_buffer[1] = RtcTime.year - 2000;
```
Comment (eku) [inline tasmota/xdrv_16_tuyamcu.ino:811]: Well, this assumes that RTC's date is behind year 2000. A better fix would be year%100.

Comment (arendst) [reply tasmota/xdrv_16_tuyamcu.ino:811]: Agree. Done. Thx.

**Comment by Bettman66** [comment]
I had taken over the solution from the file "xdsp_03_matrix.ino" in line 307, shouldn't it also be changed?

**Pull request merged**